.github/workflows/unit-tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ubuntu-latest]
-        php-versions: ['7.2','7.3', '7.4']
+        php-versions: ['7.3', '7.4', '8.0']
```

Comment:
Our [official minimal PHP version](https://de.wordpress.org/plugins/onoffice-for-wp-websites/) for the plugin is 7.2. What do you think about having `['7.2', '7.4', '8.0']`?

Reply:
But phpunit 9 requires php 7.3 or higher

Reply:
@jmaas-onoffice I think we can skip run unittest at php7.2

Reply:
Then I think you need to revert 6fc5136a46aba4b15426477f3017cacf0f81d497. We can just update the minimum PHP version and make a new major release.